src/ClusterBootstrap/template/RepairManager/rule-config.yaml

```diff
@@ -6,6 +6,9 @@ rules:
   ecc_reboot_node_rule:
     module_name: rules.ecc_reboot_node_rule
     class_name: ECCRebootNodeRule
+  monitor_smi_latency_rule:
+    module_name: rules.monitor_smi_latency_rule
```

Comment:
Can we use `nvidia_smi_latency_rule` for clarity? The corresponding class name would be `NvidiaSmiLatencyRule`